### DISCUSSION TITLE
Collabora server do not connect

### DISCUSSION THREAD

**Discussion by bigAboo80** [nextcloud/all-in-one#3348] [Questions] [answered]
HI.
I did in docker

curl -vvv https://$NC_DOMAIN:443/hosting/discovery
exit
And saw wrong server ip

$NC_DOMAIN - how can i change this variable?

**Comment by szaimen** [comment]
> wrong server ip

What did you see? Can you post the log here?

**Reply by bigAboo80** [reply]
Log from where? 
sudo docker exec -it nextcloud-aio-nextcloud bash
79c50a50424d:/var/www/html# curl -vvv https://$NC_DOMAIN:443/hosting/discovery
exit
* processing: https://nc.mscsrv.ru:443/hosting/discovery
*   Trying 192.168.50.174:443...
* connect to 192.168.50.174 port 443 failed: Host is unreachable
* Failed to connect to nc.mscsrv.ru port 443 after 3076 ms: Couldn't connect to server
* Closing connection
curl: (7) Failed to connect to nc.mscsrv.ru port 443 after 3076 ms: Couldn't connect to server
exit

192.168.50.174 - this is not correct ip of the host

**Reply by szaimen** [reply]
I see. Can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by bigAboo80** [reply]
https://yourdomain.com/settings/admin/richdocuments

Gives me 

![image](https://github.com/nextcloud/all-in-one/assets/48507036/dc7cf0d3-2344-4137-813d-9a130808a4d3)

**Reply by szaimen** [reply]
Can you post the content of your /etc/hosts file here?

**Reply by bigAboo80** [reply]
127.0.0.1       Tower localhost
54.149.176.35   keys.lime-technology.com
95.131.149.128  GTW.MYCLOUDHOME.RU
95.131.149.128  NAS

**Reply by bigAboo80** [reply]
my setup  is docker stack on unraid

**Reply by szaimen** [reply]
So this is the /etc/hosts file from unraid?

**Reply by bigAboo80** [reply]
Yes

**Reply by szaimen** [reply]
All right, do you have a local dns server?

**Reply by bigAboo80** [reply]
yes, it is asus router

**Reply by szaimen** [reply]
Then probably the asus router is rewriting your domain to 192.168.50.174?

**Reply by bigAboo80** [reply]
![image](https://github.com/nextcloud/all-in-one/assets/48507036/24a9aa3e-da00-4e51-9175-58dad4ecffcf)
Risght now there is no such address in local network

**Comment by bigAboo80** [comment]
Nope,I have bind adress for UNRAID - 192.168.50.7
may be i need to reboot unraid?

**Reply by szaimen** [reply, marked as answer]
You could try to overwrite the dns servers in daemon.json and check if that makes it work. See https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html
